crates/glaredb/src/local.rs

```diff
@@ -73,24 +74,13 @@ impl LocalSession {
                     format!("Connected to remote GlareDB server: {}", u.cyan()),
                 )
             } else {
-                let tls_conf = if opts.disable_tls {
-                    None
-                } else {
-                    match (opts.ca_cert_path.clone(), opts.domain.clone()) {
-                        (Some(ca_cert_path), Some(domain)) => Some(TlsConfig {
-                            ca_cert_path,
-                            domain,
-                        }),
-                        _ => {
-                            return Err(anyhow!(
-                                "Specify ca-cert-path and domain in --args for TLS"
-                            ));
-                        }
-                    }
-                };
+                let url: ProxyDestination = url.try_into()?;
+
+                if !opts.disable_tls {
+                    let ca_cert = reqwest::get("https://TODO").await?.text().await?;
```

Comment:
I thought we were going to hand the cert back in the handshake with cloud rather as a new field in an existing API rather than a new one.